tss-esapi/src/context.rs

```diff
@@ -106,13 +106,19 @@ impl Context {
         )?;
 
         let esys_context = unsafe { Some(Malloced::from_raw(esys_context)) };
-        Ok(Context {
+        let mut context = Context {
             esys_context,
             sessions: (None, None, None),
             _tcti_context,
             handle_manager: HandleManager::new(),
             cached_tpm_properties: HashMap::new(),
-        })
+        };
+
+        if matches!(tcti_name_conf, TctiNameConf::LibTpms { .. }) {
+            context.startup(StartupType::Clear).unwrap();
+        }
```

Comment:
@ionut-arm yeah, what I ended up doing here.

Reply:
Is there any posibility this ends up biting us in the future? I guess if we need more options for the caller we can add a parameter to the function, or even an argument for the `LibTpms` variant.

Reply:
The only option would be for tss-tpm to change its behavior.